src/lib/git.js

```diff
@@ -166,8 +171,15 @@ async function sha(dir, branch) {
   return await git(dir, "show-ref", "-s", `refs/remotes/origin/${branch}`);
 }
 
-async function rebase(dir, branch) {
-  return await git(dir, "rebase", "--quiet", "--autosquash", branch);
+async function rebase(dir, branch, remote = "upstream") {
+  return await git(
+    dir,
+    "rebase",
+    "--quiet",
+    "--autosquash",
+    "--keep-empty",
```

Comment:
I needed to add the `--keep-empty` option here to make tests work with empty commits. Otherwise the testing could get a bit more complex.